park_info.clj

```diff
@@ -0,0 +1,367 @@
+        :playground true
+        :playground_extra: "yes, by the Kinských square entrance, nearby Kinských Villa and more under construction"
+        ;---Directions---
+        :public_transport "stop: Kobrova - bus #176"
+        :GPS_coords "50° 4’ 44.26’’ N, 14° 24’ 0.21’’ N"
+        :parking true
+        :parking_extra "OC Smíchov underground parking"
+        ;---Additional Information---
+        :path_type "paved"
+        :administered_by "Prague City"
+        :opening_hours "Open year-round"
+        ;---Map---
+        :map_url "https://www.praha.eu/public/3/95/58/184325_4_kinskeho_mapa.pdf"       
+       
+    }
+    def Klamovka{
+        ;---Basic Info---
+        :food true
+        :food_extra "restaurant (www.zahradnirestaurace.cz) & club Klamovka (www.klubklamovka.cz)"
+        :toilet true
+        :toilet_extra "in the restaurant and club"
+        :POIs "Historically significant sights, art gallery, musical club with restaurant in garden, playground for small children"
+        ;---Children and Sport---
+        :bike_path true
+        :skating false
+        :sports_field true
+        :sports_field_extra: "Tennis courts and sports facilities in Sokolovna gymnasium"
+        :playground true
+        :playground_extra "by the garden restaurant"
+        ;---Directions---
+        :public_transport "stop: Klamovka - buses #149, #191, #217, - trams #4, #7, #9, #10, #58, #59"
+        :GPS_coords "50°4'17,04" N, 14°22'39,72" E"
+        :parking false
+        :parking_extra nil
+        ;---Additional Information---
+        :dogs true
+        :path_type "paved & asphalt"
+        :administered_by "Municipality Prague 5"
+        :opening_hours "Open year-round"
+        ;---Map---
+        :map_url "https://www.praha.eu/public/16/84/48/184296_4_klamovka_mapa.pdf"       
+       
+    }
+    def Ladronka{
+        ;---Basic Info---
+        :food true
+        :food_extra "restaurant Ladronka (www.ladronka.com)"
+        :toilet true
+        :toilet_extra "in the restaurant"
+        :POIs "Large area suited for free time activities and relaxation for all age categories, year-long: musical performances and social events, suited for a day-long stay"
+        ;---Children and Sport---
+        :bike_path true
+        :skating true
+        :sports_field false
+        :sports_field_extra: "Not in the park itself, but nearby: traffic playfround and Vypich Youth House"
+        :playground true
+        :playground_extra "by the restaurant and by the Ladronka bus stop"
+        ;---Directions---
+        :public_transport "
+        stop: U ladronky/ Rozýnova/ Štefkova - bus #191
+        stop: Vypich - buses #108, #174, #180, #191, #I3, - trams: #15, 22, 25, 57"
+        :GPS_coords "50°4'47.447" N, 14°21'52.445" E"
+        :parking true
+        :parking_extra "Yes, in streets Tomanova & Skokanská"
+        ;---Additional Information---
+        :dogs true
+        :path_type "paved, threshing & asphalt"
```

Comment:
Put in 'threshing floor' as opposed to 'threshing', confused the hell out of me for a bit.

Reply:
Will change.